src/relax/transform/legalize_ops.cc

```diff
@@ -385,17 +470,22 @@ class LegalizeMutator : public ExprMutator {
    * legalization function is not registered.
    */
   bool enable_warning_;
+  /*!
+   * \brief List of ops to be skipped from legalization
+   */
+  Optional<Array<String>> skip_ops_;
```

Comment:
du op.Get() to turn list of string into set of Ops that we can use hash map to do quick checking